examples/wxWebMapApp/CMakeLists.txt

```diff
@@ -19,18 +19,23 @@ add_executable(wxWebMapApp WIN32 MACOSX_BUNDLE ${SOURCES})
 
 target_compile_definitions(wxWebMapApp PRIVATE ${wxWidgets_DEFINITIONS} "$<$<CONFIG:DEBUG>:${wxWidgets_DEFINITIONS_DEBUG}>")
 
-target_include_directories(wxWebMapApp PUBLIC "../wxWebMapApp" ${wxWidgets_INCLUDE_DIRS} ${BOOST_INCLUDE_DIRS})
-#target_include_directories(IconicWebMapTest PUBLIC ${Boost_INCLUDE_DIRS})
-#target_include_directories(IconicWebMapTest PUBLIC ${EIGEN3_INCLUDE_DIRS})
-#target_include_directories(IconicWebMapTest PUBLIC ${GLFW_INCLUDE_DIRS})
-target_include_directories(wxWebMapApp PUBLIC "../../include")
+target_include_directories(wxWebMapApp PUBLIC ${wxWidgets_INCLUDE_DIRS} ${BOOST_INCLUDE_DIRS})
```

Comment:
BOOST_INCLUDE_DIRS should be removed if there is no dependency on `boost ` anymore?